bots/elasticsearch/bot.py

```diff
@@ -160,6 +161,14 @@ def __init__(self, config, started_by, last_run=None, **kwargs):
 
         self.es_client = Elasticsearch(self.es_url)
 
+    def get_most_recently_modified(self):
+        resp = self.es_client.search(
+            index=(self.es_index or settings.ELASTICSEARCH_INDEX),
+            doc_type='creativeworks',
+            body='{"sort": {"date_modified": "desc"}}'
```

Comment:
Oh shoot, I did that in my test which is probably why it was a little faster. Good catch ⚾️